react/states_and_effects/introduction_to_state.md

```diff
@@ -0,0 +1,103 @@
+### Introduction
+
+Any exciting application you build is likely to change over its lifetime. The changes could be as simple as toggling a dropdown menu or as complex as fetching data from an API. React provides primitives to manipulate the state of your apps, more specifically components, to make them dynamic. In this lesson, we will learn how to use state in React.  
```

Comment:
```suggestion
Any exciting application you build is likely to change over the time the user is exploring it. The changes could be as simple as toggling a dropdown menu or as complex as fetching data from an API. React provides primitives to manipulate the state of your apps, more specifically components, to make them dynamic. In this lesson, we will learn how to use state in React.  
```

In this initial line, I'm not sure how I feel about the wording. "change over lifetime" can refer to many things including the developmental changes, updates, etc.

Reply:
New wording looks good!